webpack.config.js

```diff
@@ -0,0 +1,112 @@
+const webpack = require("webpack");
+const path = require("path");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
+
+const mxHost = process.env.npm_package_config_mendixHost || "http://localhost:8080";
+const developmentPort = process.env.npm_package_config_developmentPort || "3000";
+
+const widgetConfig = {
+    entry: {
+        DropdownReference: "./src/DropdownReference/components/DropdownReferenceContainer.ts",
+        DropdownReferenceSet: "./src/DropdownReferenceSet/components/DropdownReferenceSetContainer.ts"
+    },
+    output: {
+        path: path.resolve(__dirname, "dist/tmp"),
+        filename: "widgets/com/mendix/widget/custom/dropdown/[name].js",
+        libraryTarget: "umd",
+        publicPath: "/"
+    },
+    devServer: {
+        port: developmentPort,
+        proxy: [ {
+            context: [ "**",
+                `!/widgets/com/mendix/widget/custom/dropdown/DropdownReference.js`,
+                `!/widgets/com/mendix/widget/custom/dropdown/DropdownReferenceSet.js`
+            ],
+            target: mxHost,
+            ws: true,
```

Comment:
Remove `ws: true,`, is cause console error.

Reply:
Done